monai/transforms/intensity/dictionary.py

```diff
@@ -1020,6 +1017,58 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
         return d
 
 
+class RandGibbsNoised(RandomizableTransform, MapTransform):
```

Comment:
Do you think you could make this into an `InvertibleTransform`?

Reply:
I think technically it is possible, but it would be a very bloated transform. The way the transform works is by masking out the father points in k-space. So to make it invertible, it would need to save a 3D image in k-space (as large as the input scan) for each scan that it is applied on.

Reply:
Yeah makes sense, the information is lost the same as it is with any other crop, just in the k-space instead.